Codecs/DAG-PB.md

```diff
@@ -32,24 +32,69 @@ message PBNode {
 ```
 
 The objects link names are specified in the 'Name' field of the PBLink object.
-This is non-standard and means that the 'Tsize' attribute of a link, and
-neither the 'Data' field nor the 'Links' field of the PBNode are available
-through the ipld query language. 
+All links in an object must either be blank or unique within the object.
 
-## Link Format
+## Pathing
 
-In DagCBOR, links are encoded using the raw-binary (identity, NUL) multibase in a
-field with a byte-string type (major type 2), with the tag 42. They are then
-put in the 'Hash' field of a PBLink. All links from a DAG-PB object are
-specified in the 'Links' array.
+TODO: See https://github.com/ipld/specs/issues/55
+TODO: Also see https://github.com/ipfs/unixfs-v2/issues/24 (specifically: the "Revise History" option).
+
+Neither go-ipfs nor js-ipfs agree so we have some room here because we're going to break something.
```

Comment:
How about for the purpose of the spec, the only required path mechanism that need be implemented is the `/Links/0/Hash` style. In js-ipld-dag-pb the names are just sugar and I'm guessing that the go implementation does the `/named` style as sugar as well and it just doesn't expose a resolution that can get into the paths array? Maybe it should and that's the consistency between the two. `/Links/named/Hash` is the JS sugar and `/named` is the Go sugar and that's just how it is.
Either way, that's a blocking issue that's going to hold this up so it needs to get resolved.

Reply:
I'd rather not leave sugar like that. We can get rid of it but `resolve(node, '/some/path')` should behave the same way everywhere.